q2_metadata/normalization/_norm_errors.py

```diff
@@ -0,0 +1,204 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016-2020, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import yaml
+
+
+class Error(Exception):
+    """Base class for exceptions encountered
```

Comment:
In general you want the first line of a docstring to be a single sentence and fit within 80 columns. Can you try re-formatting the docstrings in this file to follow that suggestion?

Reply:
okay that's very rigorous but okay